HOWTO.md

```diff
@@ -734,3 +736,31 @@ Options:
 
 Please note that in this mode there are some environment variables that are
 still required to be set by the user (`DI_SIGN_KEY_PATH`, `DI_HMAC_KEY_PATH`).
+
+## How to use Features
+  ### How to use `device_specific_store_driver` feature
```

Comment:
no need to indent the `###` to the right here, and a newline above and below too